sdk/identity/identity/src/credentials/azurePipelinesCredential.ts

```diff
@@ -149,33 +166,12 @@ export class AzurePipelinesCredential implements TokenCredential {
    * @throws CredentialUnavailableError
    */
   private ensurePipelinesSystemVars(): void {
-    if (
-      process.env.SYSTEM_TEAMFOUNDATIONCOLLECTIONURI &&
-      process.env.SYSTEM_TEAMPROJECTID &&
-      process.env.SYSTEM_PLANID &&
-      process.env.SYSTEM_JOBID &&
-      process.env.SYSTEM_ACCESSTOKEN
-    ) {
+    if (process.env.SYSTEM_OIDCREQUESTURI) {
```

Comment:
Notice how we're down to just _one_ env var? We might as well remove this function and just inline this check in the constructor itself.

If you still want to keep it, notice how this whole function can now just be rewritten as:

```ts
if (!process.env.SYSTEM_OIDCREQUESTURI) { 
    throw new CredentialUnavailableError(....);
}
```
And that's the _entire_ body of the function so we should simplify